GAE/src/org/waterforpeople/mapping/app/web/dto/DataBackoutRequest.java

```diff
@@ -145,9 +168,33 @@ protected void populateFields(HttpServletRequest req) throws Exception {
             includeDate = false;
         }
 
+        if (req.getParameter(FROM_DATE_PARAM) != null) {
+            fromDate = parseDate(req.getParameter(FROM_DATE_PARAM));
+        }
+        if (req.getParameter(TO_DATE_PARAM) != null) {
+            toDate = parseDate(req.getParameter(TO_DATE_PARAM));
+        }
+        if (req.getParameter(LIMIT_PARAM) != null) {
+            try {
+                limit = Integer.parseInt(req.getParameter(LIMIT_PARAM));
+            } catch (Exception e) {
+                limit = null;
+            }
+        }
+
         lastCollection = req.getParameter(LAST_COLLECTION_PARAM) != null
                 && "true".equals(req.getParameter(LAST_COLLECTION_PARAM));
         countryCode = req.getParameter(COUNTRY_PARAM);
     }
 
+    private static Date parseDate(String s) {
```

Comment:
I think @mtwestra mentioned that we should include the `toDate` up to `23:00:59` otherwise people get confused.